reactor-core/src/main/java/reactor/core/publisher/FluxSampleTimeout.java

```diff
@@ -74,6 +74,12 @@ public int getPrefetch() {
 		return main;
 	}
 
+	@Override
+	public Object scanUnsafe(Attr key) {
+		if (key == Attr.RUN_STYLE) return Attr.RunStyle.SYNC;
```

Comment:
missing coverage of `SampleTimeoutMain` and `SampleTimeoutOther` inners

Reply:
Fixed in commit [26552f6](https://github.com/reactor/reactor-core/pull/2123/commits/26552f67f2cfd7c10e59e60f3b905504c55b6e1d)